src/components/Cards/ImageCard/ImageCard.module.scss

```diff
@@ -1,6 +1,7 @@
 @use 'styles/variables/dimensions';
 
 $placeholder-background: #000;
+$media-background: #000;
```

Comment:
One small nit:

Maybe we could use `$placeholder-background` in both places rather than assigning two variables, or we could set `$media-background: $placeholder-background` if you kept them separate to be more explicit.